cmd/ping/udp.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
```

Comment:
@enocom can I get a sanity check here please? :smile: 

This is a pretty self contained binary, so I left everything in the `main` package.

Good? Bad? Otherwise?

Reply:
That's the beauty of Go -- small and focused when it needs to be. I think this looks good.